### PR TITLE
local-tester: support failpoints

### PR DESCRIPTION
**Pull request by heyitsanthony**

### CODE REVIEW

File: tools/local-tester/faults.sh [hunk 1]
```diff
@@ -52,14 +53,56 @@ function kill_all {
...
+
+function fp_rand_single {
+	fp=`rand_fp`
+	fp_activate ${HTTPFAIL[`expr $RANDOM % ${#HTTPFAIL[@]}`]} $fp 'panic("'$fp'")'
+}
+
+function fp_rand_all {
+	fp=`rand_fp`
+	for a in `seq ${#HTTPFAIL[@]}`; do fp_activate ${HTTPFAIL[$a]} "$fp" 'panic("'$fp'")'; done
+	sleep `wait_time`s
```
Comment (gyuho) [inline tools/local-tester/faults.sh]: what is this sleep time for?

**Comment by gyuho** [comment]
lgtm

File: tools/local-tester/faults.sh (hunk 1 above)
Comment (heyitsanthony) [inline tools/local-tester/faults.sh]: The sleep is there to give it some time for the stresser load to trip the failpoint.

Comment (gyuho) [inline tools/local-tester/faults.sh]: ok thanks.

**Comment by xiang90** [comment]
lgtm

**Pull request merged**